src/webgpu/shader/validation/extension/dual_source_blending.spec.ts

```diff
@@ -317,3 +317,246 @@ ${kStageIOValidationTests[t.params.attr].shader}
 `;
     t.expectCompileResult(kStageIOValidationTests[t.params.attr].pass, code);
   });
+
+const kUsageValidationTests = {
+  const: {
+    code: `@blend_src(0) const color = 1.2;`,
+    pass: false,
+    use_default_main_function: true,
+  },
+  override: {
+    code: `@blend_src(0) @id(0) override color : vec4f;`,
```

Comment:
Suggest using a scalar `f32` here, otherwise this may fail for the wrong reason (overrides must be scalar).